src/ZendDiagnostics/Check/MySQLCheck.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendDiagnostics\Check;
+
+use PDO;
+use ZendDiagnostics\Result;
+
+/**
+ * Ensures a connection to the MySQL server/database is possible.
+ */
+class MySQLCheck implements CheckInterface
```

Comment:
yeah this check seems to be quite generic indeed and should cover most RDBMS except for maybe SQLite .. so I think it would be better to call it PdoCheck (or PDOCheck? not sure about ZF2 naming conventions).